### PR TITLE
Add emma.ca

### PR DESCRIPTION
**Pull request by guillaume-plante**

### CODE REVIEW

**Comment by phallobst** [comment]
Please use this icon with a transparent background:
emma

**Comment by guillaume-plante** [comment]
No problem. Thank you for providing me the new icon. I updated in my commit to include it.

**Comment by phallobst** [comment]
Their [login page](https://app.emma.ca/en/login) features this information:
> Already have an account? Don't worry! From now on, we will send you an email with a security key each time you connect.

Do you have an account there in order to confirm that? It surely sounds like email 2FA to me. Unfortunately, I could not find any documentation to back me up on this.

**Comment by Carlgo11** [comment]
@phallobst could also be one factor authentication with email instead of passwords
🤔

**Comment by phallobst** [comment]
Crikey, you may be right!
> Emma undertakes to implement all necessary means to ensure the best possible access to the Emma Secure Access. You are solely responsible for the secure link serving as the access key to your account, which is for single use only.

(see https://emma.ca/en/terms-of-use 11.2)

That would be a mind-bendingly stupid idea...

**Comment by guillaume-plante** [comment]
There is no 2FA. It's simply a passwordless login. The same method has the one presented in this blog post from Okta https://www.okta.com/blog/2020/09/magic-links/

**Review by phallobst — APPROVED** [review]

File: _data/finance.yml [hunk 1]
```diff
@@ -126,6 +126,12 @@ websites:
     facebook: ditech
     img: ditech.png
 
+  - name: Emma
+    url: https://emma.ca/
+    twitter: EmmaAssurance
+    facebook: EmmaAssurance
+    img: emma.png
+
```
Comment (Carlgo11) [inline _data/finance.yml:136]:
```suggestion
    regions:
      - ca

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
What happened here? :confused:

**Comment by Carlgo11** [comment]
Actions temporarily stopped working so I pushed an empty commit to trigger it again. I did however accidentally also merge our latest commits in master as well 😅

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**